pavelib/paver_consts.py

```diff
@@ -4,7 +4,7 @@
 
 # Constants for e2e tests
 E2E_TEST_REPORT = 'e2e_tests_results.xml'
-LOG_DIR = path(os.path.join('logs')).abspath()
+LOG_DIR = path(os.path.join('log')).abspath()
```

Comment:
reviewer note: 
The jenkins job is set up to archive the "log" directory, not the "logs" directory. This was a typo.